lms/djangoapps/teams/static/teams/js/collections/team.js

```diff
@@ -1,18 +1,22 @@
 ;(function (define) {
     'use strict';
-    define(['common/js/components/collections/paging_collection', 'teams/js/models/team', 'gettext'],
-        function(PagingCollection, TeamModel, gettext) {
-            var TeamCollection = PagingCollection.extend({
+    define(['teams/js/collections/base', 'teams/js/models/team', 'gettext'],
+        function(BaseCollection, TeamModel, gettext) {
+            var TeamCollection = BaseCollection.extend({
                 initialize: function(teams, options) {
-                    PagingCollection.prototype.initialize.call(this);
+                    var self = this;
+                    BaseCollection.prototype.initialize.call(this, options);
 
-                    this.course_id = options.course_id;
-                    this.server_api['topic_id'] = this.topic_id = options.topic_id;
-                    this.server_api['expand'] = 'user';
-                    this.perPage = options.per_page;
-                    this.server_api['course_id'] = function () { return encodeURIComponent(this.course_id); };
-                    this.server_api['order_by'] = function () { return 'name'; }; // TODO surface sort order in UI
-                    delete this.server_api['sort_order']; // Sort order is not specified for the Team API
+                    this.server_api = _.extend(
+                        {
+                            topic_id: this.topic_id = options.topic_id,
+                            expand: 'user',
+                            course_id: function () { return encodeURIComponent(self.course_id); },
+                            order_by: function () { return 'name'; } // TODO surface sort order in UI
+                        },
+                        BaseCollection.prototype.server_api
+                    );
+                    delete this.server_api.sort_order; // Sort order is not specified for the Team API
```

Comment:
I'm confused about sort order and order_by. Not related to your changes though....